miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -2870,4 +2871,43 @@ private static void setId(Consumer<Long> setter, Identifiable value) {
     setter.accept(value == null ? null : lookup.apply(value));
   }
 
+  /**
+   * Converts from Runscanner's Platform to MISO's PlatformType.
+   * 
+   * @param rsType Runscanner Platform
+   * @return equivalent MISO PlatformType
+   */
+  public static PlatformType getMisoPlatformTypeFromRunscanner(ca.on.oicr.gsi.runscanner.rs.dto.type.Platform rsType) {
+    return PlatformType.valueOf(toMisoFormat(rsType.name()));
+  }
+
+  /**
+   * Converts from Runscanner's HealthType to MISO's HealthType
+   * 
+   * @param rsType Runscanner HealthType
+   * @return equivalent MISO HealthType
+   */
+  public static HealthType getMisoHealthTypeFromRunscanner(ca.on.oicr.gsi.runscanner.rs.dto.type.HealthType rsType) {
+    return HealthType.valueOf(toMisoFormat(rsType.name()));
+  }
+
+  /**
+   * Converts ABCD to Abcd for compatibility with MISO's enum formatting.
+   * 
+   * @param name Value of a Runscanner enum, all caps
+   * @return name made Upper Camel Case
+   */
+  private static String toMisoFormat(String name) {
+    return name.substring(0, 1) + name.substring(1).toLowerCase();
```

Comment:
I really think this should be a switch block. It's much easier to understand

Reply:
Or just capitalize the enum values in MISO too so this is unneccessary

Reply:
@djcooke I tried that, it broke a lot of database things so I figured this was the best route for now

@apmasell Could you please elaborate? I currently don't see how there's any decision-making in this function

Reply:
You would have to include a DB migration that updates the DB values to match the enum updates